src/css/components.css

```diff
@@ -0,0 +1,673 @@
+
+    &:focus,
+    &:hover {
+      color: var(--color-cardinal-red);
+    }
+  }
+
+  /* Form Components */
+  .button {
+    font-family: var(--font-sans);
+    cursor: pointer;
+    display: inline-block;
+    border: none;
+    font-weight: var(--font-weight-normal);
+    line-height: 1;
+    text-align: center;
+    text-decoration: none;
+    width: auto;
+    transition: background-color 0.25s ease-in-out, color 0.25s ease-in-out;
+    padding: 1rem 2rem;
+    background-color: var(--color-digital-red);
+    color: var(--color-white);
+
+    &:active,
+    &:hover,
+    &:focus {
+      text-decoration: underline;
+    }
+
+    &:hover,
+    &:focus {
+      background-color: var(--color-black);
+      color: var(--color-white);
+    }
+
+    &:focus {
+      box-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06);
+    }
+  }
+
+  .input {
+    display: block;
+    border-color: var(--color-black-20);
+    color: var(--color-black);
+    font-size: 1.8rem;
+    font-weight: var(--leading-snug);
+  }
+
+  .radio,
+  .checkbox {
+    background-color: var(--color-white);
+    border-width: 3px;
+    border-color: var(--color-digital-blue);
+    width: 2.4rem;
+    height: 2.4rem;
+  }
+
+  .radio:checked,
+  .checkbox:checked {
+    background-color: var(--color-digital-blue);
+  }
+
+  .fieldset {
+    border: 0;
+  }
+
+  .label {
+    display: block;
+    font-size: 1.8rem;
+  }
+
+  .legend {
+    font-weight: var(--font-weight-bold);
+    font-weight: var(--leading-display);
+  }
+
+  .textarea {
+    height: 16rem;
+    display: block;
+    border-color: var(--color-black-20);
+    color: var(--color-black);
+    font-size: 1.8rem;
+    font-weight: var(--leading-snug);
+  }
+
+  .select {
+    display: block;
+    border-color: var(--color-black-20);
+    color: var(--color-black);
+    font-size: 1.8rem;
+    font-weight: var(--leading-snug);
+    background-color: white;
+  }
+
+  /* Layout Components */
+  .centered-container,
+  .cc {
+    padding-left: var(--screen-margin-xs);
+    padding-right: var(--screen-margin-xs);
+    margin-left: auto;
+    margin-right: auto;
+
+    @media (min-width: 576px) {
+      padding-left: var(--screen-margin-sm);
+      padding-right: var(--screen-margin-sm);
+    }
+
+    @media (min-width: 768px) {
+      padding-left: var(--screen-margin-md);
+      padding-right: var(--screen-margin-md);
+    }
+
+    @media (min-width: 992px) {
+      padding-left: var(--screen-margin-lg);
+      padding-right: var(--screen-margin-lg);
+    }
+
+    @media (min-width: 1200px) {
+      padding-left: var(--screen-margin-xl);
+      padding-right: var(--screen-margin-xl);
+    }
+
+    @media (min-width: 1700px) {
+      padding-left: calc((100% - 1500px) / 2);
+      padding-right: calc((100% - 1500px) / 2);
+    }
+
+    .centered-container,
+    .cc {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+
+  /* Responsive grid gap */
+  .grid-gap {
```

Comment:
I could argue this is more of a utility than a component. But I won't die on that hill.

Reply:
Yeah the reason I put it in component is that you can override it with the core utilities, so grid-gap gap-y-10 for example will always work